### DISCUSSION TITLE
[Weekly Meeting] [2024-04-12] - Meeting Notes

### DISCUSSION THREAD

**Discussion by porcelli** [apache/incubator-kie-issues#1088] [Weekly Meetings]
- Community
   - SonataFlow Logo

- Apache KIE 10
  - Upgrades to Runtimes to LTS
      - Quarkus 3.8.3
         - Extension of code modification? Not that big, isolated to: DevUI (actually v3 API - already addressed by SWF and BPMN, optaplanner in PR), Hibernate more strict, Test failure with SetEnvironmentVar annotation (needs investigation)
         - Look at Native compilation might be a concern, need to be investigated. Ask the Zulip in Quarklus channel. Reflection json config file changed in the past, Gabriel has raised concerned. Need to be investigated. Expected that CI can also take on Native when we submit PR with changes. In the past this 
         - Folders where Quarkus is storing the code-gen results. We use internal API, no commitment for compatibility form Quarkus team.
      - SpringBoot 3.2.4
          - Merged!!!
      - Observation: dependency management needs some love, lot's of unused dependencies and version properties 
  - Progress on https://github.com/apache/incubator-kie-issues/issues/967
     -  Productive week, a few hiccup, missing images, 999 snapshots rogue, all sort of issues, but all has been solved. Yay!
     - Today we are...... ~50% progress mark.
     - Weekly snapshot is more reliable
     - Kogito Examples repo seems broken! @fantonangeli has a PR to fix! We need get this merged asap - green but missing reviews. What exactly? Not sure... was a comment from @Rikkola in Zulip.
     - Ballpark estimation - Last week: Week of April 22nd.

---

Recording:

https://us06web.zoom.us/rec/share/0UAyn9JCi9Nb03lkJBlLi23J5vTdJsD5aEfmG5nUU_ZvDnklIy76RDPRsN9I_Fw2.D25ni_Vdhg1ylRgq
Passcode: 5LcXL2H%

**Comment by mariofusco** [comment]
I didn't try what happens with Kogito, but this PR is enough to migrate Drools to Quarkus 3.8.3 https://github.com/apache/incubator-kie-drools/pull/5850
